### ISSUE TITLE
Using TeamEchoBot template + Outgoing Webhook - Bot replies "Sorry, there was a problem encountered with your request"

### ISSUE THREAD

**Issue by mcblair** [OfficeDev/BotBuilder-MicrosoftTeams-dotnet#9]
Using this exact project, https://github.com/OfficeDev/BotBuilder-MicrosoftTeams-dotnet/tree/master/CSharp/Samples/Microsoft.Bot.Builder.Teams.TeamEchoBot. 

Configured with a valid app id and app password + Outgoing webhook pointing to this project deployed to azure we see an immediate response: 
![image](https://user-images.githubusercontent.com/12667624/51663943-2af93580-1ff3-11e9-80da-5a7b64ff86f0.png)

I understand I have an outgoing webhook secret, and I'm guessing it is related to this issue. 

Any help?

**Comment by mcblair** [comment]
In the TeamsEchoBot readme, I see:
```
How to test
Use the sample manifest and change the botId at --YOURBOTIDHERE-- and use your bot Id.
```

Is there something obvious that is going over my head? It seems the instructions are incomplete. Simply updating the manifest is not enough to test.

**Comment by mcblair** [comment]
I have confirmed that using `App Studio` or sideloading allows this to function.

Outgoing webhooks not supported?

**Comment by RamjotSingh** [comment]
Yes Outgoing webhooks are not supported yet. Outgoing webhooks use an auth which is not supported yet.
